internal/errors/net.go

```diff
@@ -36,4 +36,8 @@ var (
 	ErrNoPortAvailable = func(host string, start, end uint16) error {
 		return Errorf("no port available for Host: %s\tbetween %d ~ %d", host, start, end)
 	}
+
+	ErrLookupIPAddrNotFound = func(host string) error {
```

Comment:
could you please add the comment about this function?

Reply:
thanks